### PR TITLE
multiple fixes

### PR DESCRIPTION
**Pull request by jazzl0ver**
Please, merge

### CODE REVIEW

File: catalog/kafkamanager/1.3.3/dockerfile/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,10 @@ RUN set -ex; \
   cd /tmp/kafka-manager; \
   git checkout ${KM_REVISION}; \
   echo 'scalacOptions ++= Seq("-Xmax-classfile-name", "200")' >> build.sbt; \
-  ./sbt clean dist; \
+  ./sbt clean dist 2>&1 | tee sbt-out.log; \
+  while $(grep -q FAILED sbt-out.log); do \
```
Comment (JuniusLuo) [inline catalog/kafkamanager/1.3.3/dockerfile/Dockerfile]: The dead loop does not look good. The sbt download issue would be a temporary issue. Could simply retry the build.

Comment (jazzl0ver) [reply catalog/kafkamanager/1.3.3/dockerfile/Dockerfile]: Yeah, I agree. I can add a counter to limit attempts to, say, 10. Will this work for you?

**Comment by jazzl0ver** [comment]
@JuniusLuo is there anything that prevents merging now?

**Comment by cloudstax** [comment]
Thanks for the fixes!

**Comment by JuniusLuo** [comment]
The kafka manager container image is updated to include the patch. You could go ahead and try it. Thanks.

**Pull request merged**